pio_examples/gui/platformio.ini

```diff
@@ -32,12 +32,20 @@ lib_ignore =
 [env:esp8266]
 platform = espressif8266
 board = nodemcuv2
+upload_port  = COM8
```

Comment:
hm... would be lovely to have these settings local to the dev environment... but its fine for now